gtest/src/log.rs

```diff
@@ -144,6 +144,14 @@ impl Log {
         log
     }
 
+    /// Create a `Log` with a reply code of `SuccessReplyReason::Auto`.
+    pub fn auto_reply_builder() -> Self {
```

Comment:
auto replies never has payload, better to somehow reflect it

Reply:
> auto replies never has payload, better to somehow reflect it

How? Add a separate `AutoReplyLog` type? Seems to be overkill.

Reply:
Why is it overkill?

Reply:
At least panic once added payload for such kind of Log

Reply:
> At least panic once added payload for such kind of Log

Sounds reasonable; I added it.